models/comments.models.js

```diff
@@ -10,4 +9,22 @@ function fetchCommentsById(params) {
     });
 }
 
-module.exports = {fetchCommentsById};
\ No newline at end of file
+function addCommentByArticleId(comment, params) {
+    if(!comment.body.length) return Promise.reject({status: 400, msg: 'empty comment'});
+    return db.query(`SELECT * FROM users WHERE username = $1`, [comment.username])
```

Comment:
So if you try and insert a comment with a user who does not exist in the database, PSQL will give you an error back, with a specific code that you can utilise in your error handling :)